test/multipart-small-stream.test.js

```diff
@@ -51,24 +50,24 @@ test('should throw fileSize limitation error on small payload', { skip: true },
 
   try {
     const [res] = await once(req, 'response')
-    t.equal(res.statusCode, 413)
+    t.assert.strictEqual(res.statusCode, 413)
     res.resume()
     await once(res, 'end')
   } catch (error) {
-    t.error(error, 'request')
+    t.assert.ok(error, 'request')
```

Comment:
@matteo-gobbo 

`t.error` is not the same as `t.assert.ok`.

https://tapjs.github.io/tapjs/classes/_tapjs_test.index.Test.html#error
> Assert that an error object is not provided.

https://nodejs.org/api/assert.html#assertokvalue-message
> Tests if value is truthy.